components/package.json

```diff
@@ -12,9 +12,9 @@
   ],
   "homepage": "https://barista.dynatrace.com",
   "peerDependencies": {
-    "@angular/cdk": "^0.0.0-NG-CDK",
-    "@angular/core": "^0.0.0-NG",
-    "@angular/common": "^0.0.0-NG",
+    "@angular/cdk": "{{VERSION}}",
```

Comment:
I'm guessing we had a reason why we kept the two versions separate? Should we do this again or do we sync these now?

Reply:
What i thought is that we always want to set the same version on the release package json as we have in our own package.json. I am taking all packages that have the placeholder in the value and take the corresponding version from the root package.json